cdap-api/src/main/java/io/cdap/cdap/api/common/RuntimeArguments.java

```diff
@@ -31,6 +31,9 @@ public final class RuntimeArguments {
   private static final String ASTERISK = "*";
   private static final String DOT = ".";
 
+  // Runtime argument to skip the normal macro evaluation during app spec regeneration.
+  public static final String SKIP_NORMAL_MACRO_EVALUATION = "system.skip.normal.macro.evaluation";
```

Comment:
I think the reason you put this in `RuntimeArguments` instead of `SystemArguments` is because it is in api module so the app can access it?

Reply:
yeah, moved to `SystemArguments` class now.